modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -343,6 +343,11 @@ up    <string name="player_sleep_in_one_chapter">Sleeping in 1 chapter</string>
     <string name="player_up_next_move_to_top">Move to top</string>
     <string name="player_up_next_select">Select</string>
     <string name="player_up_next_time_remaining">Total time remaining %s</string>
+    <string name="player_up_next_time_left">%s left</string>
```

Comment:
We already have `time_left` should we use this instead.
```
    <string name="time_left">%s left</string>
```
or we could link back to it
```
   <string name="player_up_next_time_left" translatable="false">@string/time_left</string>
```

Reply:
Good point. Linked back in [9e8d5ad](https://github.com/Automattic/pocket-casts-android/pull/2192/commits/9e8d5add81afacc7dd6084c2ff8c7c8019d75703)